CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 1.0.0-beta.15 (2017-03-22)
+
+- Images! Images stored in `src/assets` and referenced via `~/assets/` in all HTML/SCSS files are now bundled.
+- Added `preload` and `postload` plugin hooks.  Beginning initial work for a lifecycle hooks. (Thanks @Blackbaud-SteveBrush!)
+- Automatically including `envid` and `svcid` in `SkyAuthHttp` requests.
```

Comment:
The verbs that start these are usually past tense. I'd change "including" to "included."